jupyterlab_nodeeditor/yggdrasil_support.py

```diff
@@ -1,9 +1,37 @@
 import uuid
-
 import yaml
 
 from .node_editor import NodeEditor
 
+from yggdrasil.examples import yamls as ex_yamls
+import jupyterlab_nodeeditor as jlne
+import yggdrasil.yamlfile
+import re
+
+photosynthesis_model = {}
+with open(ex_yamls['fakeplant']['python'], "r") as test_model:
+    photosynthesis_model = yaml.safe_load(test_model)['model']
+    
+# Hard coded function to clean up Photosynthesis model
+def dict_conversion(model_dict):
+    model_dict.pop("language")
+    model_dict.pop("args")
+    model_dict["title"] = model_dict.pop("name")
+    for i in model_dict['inputs']:
+        i["title"] = i.pop("name")
+        i["socket_type"] = i["default_file"].pop("filetype")
+        i["key"] = "temp"
+        i.pop("default_file")
+    for i in model_dict['outputs']:
+        i["title"] = i.pop("name")
+        i["socket_type"] = i["default_file"].pop("filetype")
+        i["key"] = i["default_file"].pop("field_names")
+        i.pop("default_file")
+
+# Tesing Code
+ps = jlne.NodeEditor()
+dict_conversion(photosynthesis_model)
+ps.add_component(photosynthesis_model)
```

Comment:
I think this should be in a function:

```suggestion
def load_example(ps = None):
    ps = ps or jlne.NodeEditor()
    dict_conversion(photosynthesis_model)
    ps.add_component(photosynthesis_model)
```
I'm not super keen on the in-place modification, though -- see my comments above.